controllers/strategic-investments/index.js

```diff
@@ -1,28 +1,84 @@
 'use strict';
 const path = require('path');
 const express = require('express');
-const { concat, startsWith } = require('lodash');
+const { concat, get, set, startsWith } = require('lodash');
 
 const {
     injectBreadcrumbs,
     injectListingContent,
     injectStrategicProgramme,
-    injectStrategicProgrammes
+    injectStrategicProgrammes,
+    setCommonLocals
 } = require('../../middleware/inject-content');
 
 const router = express.Router();
 
-router.get('/', injectListingContent, injectBreadcrumbs, injectStrategicProgrammes, function(req, res) {
+router.use(injectBreadcrumbs, (req, res, next) => {
```

Comment:
I nicked this idea from the funding programmes controller, meaning that the API now [doesn't need to supply breadcrumbs](https://github.com/biglotteryfund/craft-dev/pull/177/files#diff-0c5616765ab2e0289f6948e349af3e37L25), which allows non-child pages to still get a valid section crumb (which they didn't otherwise).

Reply:
Nice, when I first added that I was half thinking that the API could supply breadcrumbs but in practice it's almost always made more sense for the app to do it. Thanks for cleaning this up.